backend/src/test/java/gov/cdc/usds/simplereport/service/TestOrderServiceTest.java

```diff
@@ -1667,9 +1658,6 @@ void getTestEventsResults_withResultFilter_returnsFluAndCovidNegatives() {
     Organization org = _organizationService.getCurrentOrganization();
     Facility facility = _dataFactory.createArchivedFacility(org, "deleted facility");
     Person p = _dataFactory.createMinimalPerson(org, facility);
-    var notExpected_allPos =
-        _dataFactory.createMultiplexTestEvent(
-            p, facility, TestResult.POSITIVE, TestResult.POSITIVE, TestResult.POSITIVE, false);
```

Comment:
The variable may not be needed, but I think we do need to create a test event that will not be returned in order to ensure the filter works.

Reply:
Ah, good catch. Instead of removing this variable I added an assertion that it wasn't present in the expected results list.